install/installer/traf_cloudera_mods

```diff
@@ -367,27 +370,37 @@ ssh -q -n $HDFS_NODE 'sudo su hdfs --command "' "$HADOOP_BIN_PATH"'/hdfs dfsadmi
 #       enabled and HDFS has been restarted
 echo "***INFO: Setting HDFS ACLs for snapshot scan support"
 ssh -q -n $HDFS_NODE 'sudo su hdfs --command "' "$HADOOP_BIN_PATH"'/hdfs dfs -mkdir -p /hbase/archive"'
-if [ $? != 0 ]; then
+errorFound=$?
+echo "ERROR Code: $errorFound"
```

Comment:
The "echo" was added earlier on to help figure out the error if any from the command. SO that's correct, the echo shodul be within the "if" condition and only if it's non zero.